packages/cli/src/wizard/wizard.js

```diff
@@ -60,6 +70,14 @@ async function runNewProjectWizard(options) {
  * @return {Promise<void>}
  */
 async function runCommand(options) {
+  const rcFilePath = resolveRcFilePath(options.config);
```

Comment:
you shouldn't need to manually do any of this either, just change 

https://github.com/GoogleChrome/lighthouse-ci/blob/2fe16195dab6a8e84995bdc66397d5efc39a4917/packages/utils/src/lighthouserc.js#L138

to do `...ci.wizard` at the beginning

Reply:
Thx i didn't see that.
Because of the collision of `wizard.serverBaseUrl` and  `upload.serverBaseUrl` should i rename the option or just add something to the documentation that the user know that he has to define the `serverBaseUrl` only once? (at upload or wizard)?

Reply:
Let's just add something to the documentation for now that the values can't be *different* but still tell folks set it in both places. In 0.4.0 I'm planning on restructuring the upload parameters to cluster by target type, they wouldn't collide in this case.